test/integration/security-rules.spec.ts

```diff
@@ -105,23 +103,40 @@ describe('admin.securityRules', () => {
     });
   });
 
+  function getRandomInt(max: number) {
```

Comment:
Move to the top where the other helper functions are.

Reply:
Done.